js/adapt-languagePicker.js

```diff
@@ -57,6 +57,14 @@ define([
   }
 
   function showLanguagePickerView () {
+
+    if (languagePickerModel.get('_showOnCourseLoad') === false) {
+      languagePickerModel.setDefaultLanguage(languagePickerModel.get('_defaultLanguage'));
+      loadLanguage();
```

Comment:
The action to switch language has now changed. The loading of the course is now automated whenever `languagePickerModel.setLanguage` is called. See [here](https://github.com/adaptlearning/adapt-contrib-languagePicker/blob/master/js/adapt-languagePicker.js#L46) for an example.

I would also advise moving the `_showOnCourseLoad` logic out of this function to where it's called from (currently `onOfflineStorageReady`).